patterns/pattern-1/analytics-service.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: wso2ei-analytics-service
+spec:
+  type: NodePort
+  selector:
+    deployment: wso2ei-analytics
+  ports:
+  - name: 'servlet-http'
+    port: 9764
```

Comment:
Same as above, might be better to use default ports to make easier for users to understand the deployment.